requirements/pip.txt

```diff
@@ -8,7 +8,7 @@ wheel==0.37.1
     # via -r requirements/pip.in
 
 # The following packages are considered to be unsafe in a requirements file:
-pip==22.2.2
+pip==22.3
     # via -r requirements/pip.in
-setuptools==65.4.1
+setuptools==65.5.0
```

Comment:
No breaking change in the new version https://setuptools.pypa.io/en/stable/history.html#v65-5-0